env-prep/init/redis.conf

```diff
@@ -0,0 +1,3 @@
+maxmemory 1500mb
```

Comment:
Do we need that much memory for local dev environment?

The way it's used even 64 MB feels too much.